### ISSUE TITLE
sonarr season search fails parsing if series has xem data but not covering that season

### ISSUE THREAD

**Issue by thezoggy** [Sonarr/Sonarr#4849]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

episode search works regardless if the series has xem data or not. issue is only with season search.

season search for a series without xem data, looks to be fine. 

if the series has xem data, this is where it may or may-not work.
if the series has xem data for the season (scene), season search works. if no data, season results fail to get parsed and result in:
"Unknown episode or series" / "Unable to identify correct episode(s) using release name and scene mappings"

This was reported from user in discord for blackish, and confirmed once removing xem info it worked. (xem info was no longer needed).

I tested this myself and could confirm with the two series that I have in my instance that also have xem data: "We Bare Bears" and "Top Gear (US)". 

We Bare Bears - https://thexem.info/xem/show/4159
has xem data for tvdb+scene s03-04
- season search for s02 - borked
- season search for s04 - worked fine

Top Gear (US) - https://thexem.info/xem/show/3164
has xem data for tvdb s04+s06 and scene s05+s08 - not scene s06
- season search for s01 - borked
- season search for s06 - borked

### Expected Behavior

season/episode search should work the same regardless of xem data?

### Steps To Reproduce

1. season search for a series that has xem data - do season not covered vs one that is

### Environment

```markdown
- OS: Windows 10
- Sonarr: 3.0.6.1432
- Docker Install: No
- Using Reverse Proxy: No
- Browser: Firefox 96
```

### What branch are you running?

Develop

### Trace Logs?

see discord for logs / ask me if you need them

### Anything else?

more info in discord

**Comment by bakerboy448** [comment]
[Discord Discussion around/above](https://discord.com/channels/383686866005917708/383688189941907459/930604182992453703)

**Comment by markus101** [comment]
For future reports please include the logs in the issue, a link to a conversation in Discord is not sufficient.
